projects/1.18/assets/cammies-minecart-tweaks/minecarttweaks/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+	"minecarttweaks.cant_link_to_engine": "你不能隔一个链接矿车！",

```

typo
此外，`链接`还是`连接`更合适一点？
```suggestion
	"minecarttweaks.cant_link_to_engine": "你不能隔一格链接矿车！",
```

---

这里的“隔一个”指的就是连接的两个矿车之间不能隔一个矿车。

---

“链接”已改为“连接”

---

> 这里的“隔一个”指的就是连接的两个矿车之间不能隔一个矿车。

好吧，望文生义了
不过目前这个表述似乎不是很清楚？
`不能连接不相邻的矿车`？（虽然我这句话好像也不是很清楚）

---

好的，已经改正。https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/2379/commits/b9a06d5e7308628d478d50f9c03f917b9d7ff2dc